docs/multi_process.md

```diff
@@ -78,9 +78,9 @@ jax.distributed.initialize(coordinator_address="192.168.0.1:1234",
                            process_id=0)
 ```
 
-On Cloud TPU and Slurm environments, you can simply call {func}`jax.distributed.initialize()` with no
+On Cloud TPU, Slurm and Open MPI environments, you can simply call {func}`jax.distributed.initialize()` with no
 arguments. Default values for the arguments will be chosen automatically.
-When running on GPUs with Slurm, it is assumed that one process is started per GPU, i.e. each process will
+When running on GPUs with Slurm and Open MPI, it is assumed that one process is started per GPU, i.e. each process will
```

Comment:
It would be worth documenting which of SLURM or OpenMPI has precedence, since both are often used in combination with each other.

Reply:
Correct, I could clarify that, the "OpenMPI" initialization will only be used and also have higher precedence over Slurm when processes are launched via `mpirun` or `mpiexec` which will set `OMPI_MCA_orte_hnp_uri`. When launching application processes via `srun` with or without pmi/pmix feature, the Slurm initialization will be used.

Reply:
Ok good imho.
I didn't know that `mpirun` / `mpiexec` would set `OMPI_MCA_orte_hnp_uri`. If you have a link to OpenMPI's doc explaining that (if any), that would be worth adding somewhere.

Reply:
I figured it out by experimenting with it.

Reply:
OK. Spelling it out in the code as a comment next to `_ORTE_URI = 'OMPI_MCA_orte_hnp_uri'` could be a good idea imho.